package.json

```diff
@@ -30,7 +31,7 @@
 		"postinstall": "lerna bootstrap --hoist",
 		"precommit": "npm run lint:autofix && git add . -u",
 		"lint": "eslint packages/*/src/**/*.js packages/*/test/**/*.js",
-		"lint:autofix": "npm run lint -- --fix && lerna run lint:autofix:clang",
+		"lint:autofix": "npm run lint -- --fix && lerna run lint:autofix:clang && css-beautify packages/**/src/less/*.less -r && html-beautify packages/**/src/html/*.html -r",
```

Comment:
Can we split these into parallel tasks using `npm-run-all` ??

Reply:
Can and should and done.